pytensor/link/jax/dispatch/pad.py

```diff
@@ -0,0 +1,20 @@
+import jax.numpy as jnp
+
+from pytensor.link.jax.dispatch import jax_funcify
+from pytensor.tensor.pad import Pad
+
+
+fixed_kwargs = {"reflect": ["reflect_type"], "symmetric": ["reflect_type"]}
+
+
+@jax_funcify.register(Pad)
+def jax_funcify_pad(op, **kwargs):
```

Comment:
Do we need a jax dispatch? Is this because of OFG and JAX not having it implemented yet?

Reply:
No, we don't. I did it because jax has a pad implementation, so it seemed easy enough to just overload to theirs. If I remove the OFG wrapper, it should overload by default without issue.

Reply:
Even with OFG it should be fine, just put the dispatch in this PR if we haven't merged the alloc_diag yet

Reply:
Re: odd make it raise NotImplementedError with a message to open an issue here if they need it? Then well know if there's any demand (I suspect not)